TruckMuncher/MapViewController.swift

```diff
@@ -162,12 +162,10 @@ class MapViewController: UIViewController,
         var clusterAnnotation = view.annotation as? CCHMapClusterAnnotation
         var truckLocationAnnotation = clusterAnnotation?.annotations.allObjects[0] as? TruckLocationAnnotation
         
-        
         if let tappedTruckIndex = truckLocationAnnotation?.index {
             truckCarousel.currentItemIndex = tappedTruckIndex
             truckCarousel.scrollToItemAtIndex(tappedTruckIndex, animated: true)
             
-            centerMapOverCoordinate(truckLocationAnnotation!.coordinate)
```

Comment:
Sorry, i thought you were talking about the instance in carouselCurrentItemIndexDidChange() b/c that's what GitHub showed me when I clicked that link. 

No, this call is no longer needed.